frontend/packages/dev-console/src/components/hpa/HPAUtilizationField.tsx

```diff
@@ -22,6 +23,7 @@ const HPAUtilizationField: React.FC<HPAUtilizationFieldProps> = ({
   onUpdate,
   type,
 }) => {
+  const { t } = useTranslation();
```

Comment:
@rebeccaalpert build failure, looks like `const { t }` is defined above (merge issue?)

```bash
ERROR in ./packages/dev-console/src/components/hpa/HPAUtilizationField.tsx 15:12
Module parse failed: Identifier 't' has already been declared (15:12)
File was processed with these loaders:
 * ./node_modules/cache-loader/dist/cjs.js
 * ./node_modules/thread-loader/dist/cjs.js
 * ./node_modules/ts-loader/index.js
You may need an additional loader to handle the result of these loaders.
|     const thisErrorMetric = (_e = (_d = (_c = errors.formData) === null || _c === void 0 ? void 0 : _c.spec) === null || _d === void 0 ? void 0 : _d.metrics) === null || _e === void 0 ? void 0 : _e[index];
|     const error = (_g = (_f = thisErrorMetric === null || thisErrorMetric === void 0 ? void 0 : thisErrorMetric.resource) === null || _f === void 0 ? void 0 : _f.target) === null || _g === void 0 ? void 0 : _g.averageUtilization;
>     const { t } = useTranslation();
|     return (React.createElement(FormGroup, { fieldId: `${type}-utilization`, label: t('devconsole~{{label}} Utilization', { label }), helperText: t('devconsole~{{label}} request and limit must be set before {{label}} utilization can be set.', { label }), helperTextInvalid: error, validated: error ? 'error' : null },
|         React.createElement(InputGroup, null,
```

Reply:
Merge issue! :(

I'll fix it. Thanks!